src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  min-width: 1024px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.menu {
+  width: fit-content;
+  margin-right: 50px;
+}
+
+.nav_item {
+  margin-right: 20px;
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
+
+.nav_list {
+  display: flex;
+  padding: 23px 0;
+  margin: 0;
+  padding-inline-start: 0;
```

Comment:
why do you need this?

Reply:
padding and margin because there are default brower's styles. Another padding (padding: 23px 0) I already rescheduled